gallery/src/components/NxBinaryDonutChart/NxBinaryDonutChartPage.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+import NxBinaryDonutChartMinimalExample from './NxBinaryDonutChartMinimalExample';
+import NxBinaryDonutChartNoHoleExample from './NxBinaryDonutChartNoHoleExample';
+import NxBinaryDonutChartLargeHoleExample from './NxBinaryDonutChartLargeHoleExample';
+import NxBinaryDonutChartSkewedExample from './NxBinaryDonutChartSkewedExample';
+
+import './NxBinaryDonutChartPage.scss';
+
+const nxBinaryDonutChartMinimalExampleCode = require('!!raw-loader!./NxBinaryDonutChartMinimalExample').default;
+const nxBinaryDonutChartNoHoleExample = require('!!raw-loader!./NxBinaryDonutChartNoHoleExample').default;
+const nxBinaryDonutChartLargeHoleExample = require('!!raw-loader!./NxBinaryDonutChartLargeHoleExample').default;
+const nxBinaryDonutChartSkewedExample = require('!!raw-loader!./NxBinaryDonutChartSkewedExample').default;
+
+const NxBinaryDonutChartPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        <code className="nx-code">NxBinaryDonutChart</code> represents a binary donut chart.
+      </p>
+    </GalleryDescriptionTile>
+
+    <GalleryExampleTile title="Minimal Example"
+                        className="nx-binary-donut-chart-examples"
+                        codeExamples={nxBinaryDonutChartMinimalExampleCode}
+                        liveExample={NxBinaryDonutChartMinimalExample}>
+        A minimal example of a <code className="nx-code">NxBinaryDonutChart</code>.
+    </GalleryExampleTile>
+
+    <GalleryExampleTile title="Example with no Hole"
+                        className="nx-binary-donut-chart-examples"
+                        codeExamples={nxBinaryDonutChartNoHoleExample}
+                        liveExample={NxBinaryDonutChartNoHoleExample}>
+        An example of a <code className="nx-code">NxBinaryDonutChart</code> without a hole i.e. a pie chart.
+    </GalleryExampleTile>
+
+    <GalleryExampleTile title="Example with a Large Hole"
+                        className="nx-binary-donut-chart-examples"
+                        codeExamples={nxBinaryDonutChartLargeHoleExample}
+                        liveExample={NxBinaryDonutChartLargeHoleExample}>
+        An example of a <code className="nx-code">NxBinaryDonutChart</code> with a large hole.
+    </GalleryExampleTile>
+
+    <GalleryExampleTile title="Skewed Example"
+                        codeExamples={nxBinaryDonutChartSkewedExample}
+                        liveExample={NxBinaryDonutChartSkewedExample}>
+        An example of a <code className="nx-code">NxBinaryDonutChart</code> which is skewed.
```

Comment:
While this capability to do this follows naturally from the ability to pass arbitrary svg attributes, it doesn't necessarily seem like something anyone would ever be likely to really want to do, so I'm not sure it needs an example.  At the least, if it is kept then maybe it should be called out that the point of the example is to demonstrate that arbitrary attributes can be passed.

Even then though, I worry that this example might mislead people into thinking that size adjustments to this component should generally be done using props, when that isn't the case and we'd want them to use svg for that like they would on any other element.

Reply:
> it doesn't necessarily seem like something anyone would ever be likely to really want to do, so I'm not sure it needs an example

Removed this example https://github.com/sonatype/sonatype-react-shared-components/pull/172/commits/0b2fad02e32cf9927810950d59c52abe24ac9a2f